apps/src/music/types.ts

```diff
@@ -23,7 +26,8 @@ export interface MusicLevelData extends ProjectLevelData {
   toolboxDefinition?: GoogleBlockly.utils.toolbox.ToolboxInfo;
   validationTimeout?: number;
   aiCodeGenerate?: boolean;
-  aiCodeGenerateAdlib?: string;
+  aiCodeGenerateAdlibId?: string;
+  aiCodeGenerateAdlib?: AdlibType;
```

Comment:
Mind adding a few comments here so we can keep track of what is what as we're iterating on format?